Dockerfile

```diff
@@ -100,6 +100,10 @@ RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
  jq \
  && rm -rf /var/lib/apt/lists/*
 
+# Download lassie
+RUN curl -L 'https://github.com/filecoin-project/lassie/releases/download/v0.3.1/lassie-v0.3.1-linux-386.tar.gz' -o lassie.tar.gz \
```

Comment:
Since we support arm64 as well, it would be best to get `https://github.com/filecoin-project/lassie/releases/download/v0.3.1/lassie-v0.3.1-linux-$(uname -m).tar.gz`. To do this you probably need:

```
RUN sh -c 'curl -L "https://github.com/filecoin-project/lassie/releases/download/v0.3.1/lassie-v0.3.1-linux-$(uname -m).tar.gz" -o lassie.tar.gz \
  && tar -C /usr/bin -xzf lassie.tar.gz'
```

Reply:
Or [even better](https://nielscautaerts.xyz/making-dockerfiles-architecture-independent.html) to use `TARGETPLATFORM` and parse that to get the right value depending on the target build.

Reply:
great point, thanks